docs/userguide/security-governance/working-with-kyverno.md

```diff
@@ -4,313 +4,350 @@ title: Working with Kyverno
 
 [Kyverno](https://github.com/kyverno/kyverno), a [Cloud Native Computing Foundation](https://cncf.io/) project, is a policy engine designed for Kubernetes. It can validate, mutate, and generate configurations using admission controls and background scans. Kyverno policies are Kubernetes resources and do not require learning a new language. Kyverno is designed to work nicely with tools you already use like kubectl, kustomize, and Git.
 
-This document gives an example to demonstrate how to use the `Kyverno` to manage policy.
+This document gives an example to demonstrate how to use `Kyverno` to manage policies across multiple clusters.
 
-## Prerequisites
 ## Setup Karmada
 
-You just need to clone Karmada repo, and run the following script in Karmada directory. 
+To start up Karmada, you can refer to [here](../../installation/installation.md).
+If you just want to try Karmada, we recommend building a development environment by ```hack/local-up-karmada.sh```.
 
-```console
+```sh
+git clone https://github.com/karmada-io/karmada
+cd karmada
 hack/local-up-karmada.sh
 ```
 
 ## Kyverno Installations
 
 In this case, we will use Kyverno v1.6.3. Related deployment files are from [here](https://github.com/kyverno/kyverno/blob/release-1.6/config/install.yaml).
```

Comment:
still use v1.6.3?

Reply:
The changes are the same for different versions. Not sure we should update docs if kyverno releases a new version.

Reply:
We don't need to test and update the doc for each kyverno release. 
But we need to describe the correct version that was used to write this doc.

Reply:
Yes, the following Yaml is an example given based on version 1.6.3. The version is correct with the corresponding Yaml file. I think not need to update all of them to 1.8.0.